app/models/concerns/dates_in_past_validator.rb

```diff
@@ -1,10 +1,18 @@
 class DatesInPastValidator < ActiveModel::Validator
   def validate(record)
     if record.instance_of?(Era) && record.start_date && record.end_date
-      record.errors.add(:start_date, 'cannot be in the future') if record.start_date > Date.today
-      record.errors.add(:end_date, 'cannot be in the future') if record.end_date > Date.today
+      era(record)
     elsif record.instance_of?(Event) && record.date
-      record.errors.add(:date, 'cannot be in the future') if record.date > Date.today
+      event(record)
     end
   end
+
```

Comment:
Should these helpers be private?

Reply:
Should these be added to a permit type method, to only allow dates and no other types of data to be passed into our method?

More of a security feature thought.